dapr/proto/runtime/v1/dapr.proto

```diff
@@ -332,24 +332,16 @@ message BulkPublishRequestEntry {
 
 // BulkPublishResponse is the message returned from a BulkPublishEvent call
 message BulkPublishResponse {
-  // The publish statuses for different events in the BulkPublishEvent call
-  repeated BulkPublishResponseEntry statuses = 1;
+  // The entries for different events that failed publish in the BulkPublishEvent call
+  repeated BulkPublishResponseFailedEntry failedEntries = 1;
```

Comment:
Just confirming this has not been released yet, right? So we're not modifying a released proto?

Reply:
only nightly builds will be brone ... and BulkPubsub as a feature was not released in 1.9 ... 

See https://github.com/dapr/dapr/blob/release-1.9/dapr/proto/runtime/v1/dapr.proto

Reply:
Perfect.